README.md

```diff
@@ -0,0 +1,10 @@
+### Bash script to change dates and rename files
+
+remove_duplicate_files_with_paren1paren.sh
```

Comment:
A minor thing, but you can make this into a code block that will be rendered in monospaced font by surrounding it with triple-backticks like this:

> ```
> remove_duplicate_files_with_paren1paren.sh
> ```

Reply:
Drat, you can't see the triple backticks.

Reply:
@kcharter Is it standard convention to put filenames in a code block?